test/verify/check-multi-os

```diff
@@ -59,9 +59,12 @@ class TestMultiOS(MachineCase):
         self.assertEqual(result, "hi\n")
 
     def check_dbus(self, b, address):
+        # HACK: Firefox cannot do `cockpit.dbus.proxy.call` as it returns promise
```

Comment:
This message seems wrong then as well. But if tests pass, lets not block on that.